### PR TITLE
Update rc_transmitter_receiver.md

### PR DESCRIPTION
**Pull request by Benoit3**
Detail how to connect Graupner Receiver with usage SUMD protocol

### CODE REVIEW

File: en/getting_started/rc_transmitter_receiver.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ In addition to the transmitter/receiver pairs being compatible, the receiver mus
 As general guidance, receivers connect to the flight controller using the port appropriate to their supported protocol:
 
 - Spektrum and DSM receivers must connect to a **SPKT/DSM** input.
+- Graupner receiver SUMD ouput must connect to a **SPKT/DSM** input.
```
Comment (hamishwillee) [inline en/getting_started/rc_transmitter_receiver.md:100]: Typo "ouput".

@Benoit3 Thank you. Can you please confirm how this works? I Think you saying that:
- the Graupner receivers support SUMD output, and that PX4 can read this if you connect to the SPKT/GSM input. Is that correct?
- Is it also correct you can't connect to other inputs (e.g. RC In)
- Does this work for any of the Graupner receivers, or is it some subset?

Comment (hamishwillee) [reply en/getting_started/rc_transmitter_receiver.md:100]: Hi @Benoit3 - did you see my questions ^^^

**Comment by Benoit3** [comment]
Hello,

sorry to not have answered quickly, but I don't forget. Will do it today or tomorrow.

Regards,
Benoît 

Le 6 avril 2020 01:23:37 GMT+02:00, Hamish Willee <notifications@github.com> a écrit :
>@hamishwillee commented on this pull request.
>
>
>
>> @@ -97,6 +97,7 @@ In addition to the transmitter/receiver pairs being
>compatible, the receiver mus
>As general guidance, receivers connect to the flight controller using
>the port appropriate to their supported protocol:
> 
> - Spektrum and DSM receivers must connect to a **SPKT/DSM** input.
>+- Graupner receiver SUMD ouput must connect to a **SPKT/DSM** input.
>
>Typo "ouput".
>
>@Benoit3 Thank you. Can you please confirm how this works? I Think you
>saying that:
>- the Graupner receivers support SUMD output, and that PX4 can read
>this if you connect to the SPKT/GSM input. Is that correct?
>- Is it also correct you can't connect to other inputs (e.g. RC In)
>- Does this work for any of the Graupner receivers, or is it some
>subset?
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/PX4/px4_user_guide/pull/713#pullrequestreview-387873465

-- 
Envoyé de mon appareil Android avec Courriel K-9 Mail. Veuillez excuser ma brièveté.

**Comment by Benoit3** [comment]
Yes and they are right.
Will answer asap.
Benoît 

Le 6 avril 2020 01:23:37 GMT+02:00, Hamish Willee <notifications@github.com> a écrit :
>@hamishwillee commented on this pull request.
>
>
>
>> @@ -97,6 +97,7 @@ In addition to the transmitter/receiver pairs being
>compatible, the receiver mus
>As general guidance, receivers connect to the flight controller using
>the port appropriate to their supported protocol:
> 
> - Spektrum and DSM receivers must connect to a **SPKT/DSM** input.
>+- Graupner receiver SUMD ouput must connect to a **SPKT/DSM** input.
>
>Typo "ouput".
>
>@Benoit3 Thank you. Can you please confirm how this works? I Think you
>saying that:
>- the Graupner receivers support SUMD output, and that PX4 can read
>this if you connect to the SPKT/GSM input. Is that correct?
>- Is it also correct you can't connect to other inputs (e.g. RC In)
>- Does this work for any of the Graupner receivers, or is it some
>subset?
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/PX4/px4_user_guide/pull/713#pullrequestreview-387873465

-- 
Envoyé de mon appareil Android avec Courriel K-9 Mail. Veuillez excuser ma brièveté.

**Comment by Benoit3** [comment]
hello,

I apologize to not have answered earlier, I had some work this week :
- yes some of Graupner reveiver support SUMD output, but I tested only
GR16- for GR16 SUMD output is on servo socket 8, and the SUMDOF  ( in
case of loss of signal, SUMD transmission is stopped) shall be choosen
to have proper failsafe behaviour
Do not hesitate if you need more info.
Regards,Benoit

**Comment by hamishwillee** [comment]
Hi @Benoit3 

1. In that case I think the comment is sufficient :-)
2. Regarding the need to set an RC loss failsafe I've added a section "Set Signal-Loss Behaviour". That isn't specific to Graupner since it is done differently on every receiver.
3. At some point we might have individual docs for connecting/using different receivers, but this is more than enough for now. 

Thanks very much!

**Pull request merged**